watch-history-exporter-for-amazon-prime-video.js

```diff
@@ -101,13 +121,14 @@
 					// Loop over episodes watched for each series
 					for (const episodeSection of episodeSections) {
 						const episodeTitle = episodeSection?.textContent?.trim();
-
+						if (!episodeTitle) { continue; }
```

Comment:
Is `title` or `episodeTitle` ever actually empty/undefined? I tried the code with and without these lines, and the output is identical.

Reply:
Turns out this happens when running the script more than once. I've reverted the changes for this and made a separate bug report (#5) so we can fix the actual issue there.